### ISSUE TITLE
Disable fanart.tv per widget

### ISSUE THREAD

**Issue by skyfsza** [jurialmunkey/plugin.video.themoviedb.helper#25]
Would it be possible to have an option where I can disable fanart.tv on a per widget basis? similar to how you can remove &widgets=True on an individual widgets for the default action?

Use case would be having widgets within VideoInfo (similar to your AZ2 tmdb widgets) without fanart.tv so as to speed up load times, but still having fanart.tv as default for home / all other widgets where Clearlogo etc would be useful.

**Comment by jurialmunkey** [comment]
Yeah this is a good idea. I was having a similar thought the other day - just thinking about the best way to implement it

**Comment by jurialmunkey** [comment]
Okay, in latest update now only additional artwork is pulled from fanart.tv when inside the add-on itself. There is an additional option in settings to enable artwork for widgets (i.e. this setting will enable fanarttv for plugin paths that have `&widget=True`).

If you need to use additional artwork elsewhere, you can force it by adding `&amp;fanarttv=True` - however, the user will still need enable fanart.tv in settings (i.e. you can't override user setting to disable fanart.tv).

**Comment by drinfernoo** [comment]
So by adding `&amp;fanarttv=True` to a widget item *without* `&widget=True`, we could have a widget that gets extra artwork, but also follows the default action?

**Comment by jurialmunkey** [comment]
Yep, anything you add `&amp;fanarttv=True` to will get extra artwork (as long as it isn't disabled in addon settings)

**Comment by drinfernoo** [comment]
@jurialmunkey I'm not seeing this behavior. With a widget with only `&amp;widget=True`, it looks like I'm getting extended artwork, but when removing that tag and replacing it with `&amp;fanart=True`, I don't.

**Comment by jurialmunkey** [comment]
> 
> 
> `&amp;fanart=True`

It needs to be `&amp;fanarttv=True`
You're missing the "tv" part of "fanarttv"

**Comment by drinfernoo** [comment]
@jurialmunkey Haha, wow... let me try again 😅

**Comment by drinfernoo** [comment]
Ok, this is working! Nothing to see here 😆

**Comment by skyfsza** [comment]
Thanks for changes, can confirm working perfectly for me.
